### DISCUSSION TITLE
求助。电脑版安装黑屏

### DISCUSSION THREAD

**Discussion by SkyStyle123** [localsend/localsend#1240] [L-简体中文]
![360截图20240328005223420](https://github.com/localsend/localsend/assets/165194239/9ed3c074-f4b2-48f4-93b2-b21531d897ec)

WIN7系统 64位。安装最新的1.14版，点开后一直黑屏，试了老版本也是如此。 有大神支招吗？非常感谢！

**Comment by Martinxux** [comment]
你在后台开了两个进程就会这样
